.travis.yml

```diff
@@ -28,10 +28,10 @@ install:
   - docker run -dit --name fwk fwk  # Run it detached, with stdin open and TTY to keep it running
   - docker ps -a
 script:
-  - docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/dal/tests/test_alba.py"
-  - docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/dal/tests/test_hybrids.py"
-  - docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/lib/tests/alba_tests/test_lib_alba.py"
-  - docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/lib/tests/alba_tests/test_nsmcheckup.py"
-  - docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/webapps/api/tests/test_api_configuration.py"
+  - docker exec -it fwk bash -c "ovs unittest test_alba"
```

Comment:
The idea of the change was to change all test files for this specific repo to test_alba_ ...
That way instead of 
- `docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/lib/tests/alba_tests/test_lib_alba.py"` 
- `docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/dal/tests/test_alba.py"`
- `docker exec -it fwk bash -c "ovs unittest /opt/OpenvStorage/ovs/lib/tests/alba_tests/test_nsmcheckup.py" `

we could write `docker exec -it fwk bash -c "ovs unittest test_alba_"`

Reply:
This wont work. 
A list of short names is composed, and the `test_alba_` is checked for presence in this list (which will result in null)
Only afterwards `test.startswith(filename)` is checked.

Reply:
Make it work :)